package.json

```diff
@@ -33,7 +33,8 @@
     "redis": "2.6.3",
     "reflect-metadata": "0.1.10",
     "request": "2.81.0",
-    "rolling-rate-limiter": "0.1.9"
+    "rolling-rate-limiter": "0.1.9",
+    "typemoq": "^2.0.1"
```

Comment:
May be replace sinon mocks by typemoq for uniformity?

Reply:
I'm not sure what I like more yet :) TypeMoq was specially designed for Typescript but produces too much code, however I was got stuck with sinon - wasn't able to setup mocks due to failing type checks, I will try sinon 1 more time later.

Reply:
Just move it to it's own issue :)